src/tests/ftest/launch.py

```diff
@@ -113,7 +116,7 @@ def ignore_unknown(self, node):  # pylint: disable=no-self-use,unused-argument
     if os.path.isfile(yaml_file):
         with open(yaml_file, "r", encoding="utf-8") as open_file:
             try:
-                yaml_data = yaml.load(open_file.read(), Loader=DaosLoader)
+                yaml_data = yaml.load(open_file.read(), Loader=DaosLoader)  # nosec
```

Comment:
Could we just use `yaml.safe_load` here?

Reply:
The `DaosLoader` inherits `yaml.SafeLoader`